src/main/java/com/cs3250p1/project1/Main.java

```diff
@@ -4,9 +4,16 @@
 
```

Comment:
Improved speed of bulk order processing is nice. Communication with sql table is good. Supplier order is good as it is, I think that won't need changing and we can just go wth it